public/tracker.js

```diff
@@ -115,9 +135,27 @@ function addItem(event) {
   });
 };
 
-$.ajax({
-  cache: false,
-  dataType: 'json',
-  url: '/api/data',
-  success: updateView
-});
\ No newline at end of file
+$(document).ready(function(){
+  var url = window.location.href;
+  var idLocation = url.indexOf('#');
+  if (idLocation == -1) {
+      $.ajax({
+        cache: false,
+        dataType: 'json',
+        url: '/api/data',
+        success: updateView
+      });
+  }
+  else {
+    var gameID = url.substring(idLocation + 1);
```

Comment:
You should verify gameID is not empty.  Instead of doing 
```
if (idLocation === -1) {
}
```
do 
```
if (gameID.length > 0) {
  //...
}
```

and compute gameID above